lsix

```diff
@@ -68,7 +68,12 @@ fi
 if [[ "$COMSPEC" ]]; then
     alias convert="magick convert" # Shun MS Windows' "convert" command.
 fi
-    
+
+# convert has been deprecated with ImageMagick 7
+if [ "$(identify -version | head -n 1 | awk '{print $3}' | cut -d'.' -f1)" -ge 7 ]; then
```

Comment:
I like that you were thorough, but parsing version numbers like this is notoriously brittle. Also, while I make pipelines like that at the command line all the time, in a shell script I'd prefer to take a moment to avoid forking quite so many processes. For example: `$(identify -version | awk 'NR==1 { print $3 >= 7 }'`.

Another problem is that for many years ImageMagick 7 had all the same executables as IM6 before deprecating `convert`. Will `identify` be deprecated someday? All we know for sure is that IM6 does not have `magick` and IM7 does.